packages/import-cost/src/webpack.js

```diff
@@ -15,14 +15,12 @@ function getEntryPoint(packageInfo) {
 
 function calcSize(packageInfo, callback) {
   const entryPoint = getEntryPoint(packageInfo);
-  const modulesDirectory = path.join(pkgDir.sync(path.dirname(packageInfo.fileName)), 'node_modules');
-
-  const externals = {
-    react: 'React',
-    'react-dom': 'ReactDOM',
-    lodash: '_'
-  };
-  delete externals[packageInfo.name];
+  const packageRootDir = pkgDir.sync(path.dirname(packageInfo.fileName));
+  const modulesDirectory = path.join(packageRootDir, 'node_modules');
+  const importedPkg = require(path.join(modulesDirectory, packageInfo.name, 'package.json'));
```

Comment:
Assumes the package is accessible from the top level of node_modules. I think this is OK, but open to other suggestions.